src/analysis/convex/cone.lean

```diff
@@ -196,8 +196,7 @@ the order relation is the one defined by the cone.
 -/
 def to_ordered_semimodule {M : Type*} [ordered_add_comm_group M] [semimodule ℝ M]
   (S : convex_cone M) (h : ∀ x y : M, x ≤ y ↔ y - x ∈ S) : ordered_semimodule ℝ M :=
-{ to_semimodule := ‹_›,
-  .. (show ordered_semimodule ℝ M, from ordered_semimodule.mk'
+{ .. (show ordered_semimodule ℝ M, from ordered_semimodule.mk'
```

Comment:
The trick with `..` was here to avoid marking this `def` as `noncomputable`. Please make it a `lemma` and use `ordered_semimodule.mk'` directly.